delta/sourcing/src/main/scala/ch/epfl/bluebrain/nexus/delta/sourcing/projections/tracing/ProgressTracingConfig.scala

```diff
@@ -0,0 +1,10 @@
+package ch.epfl.bluebrain.nexus.delta.sourcing.projections.tracing
+
+/**
+  * Config to push stream progress metrics in Kamon
+  * @param prefix
+  *   the prefix for the different metrics
+  * @param tags
+  *   the tags
+  */
+final case class ProgressTracingConfig(prefix: String, tags: Map[String, Any])
```

Comment:
This is not tracing, but kamon metrics... maybe we should rename this to `ProgressKamonMetricsConfig` along with the other fn names.

Reply:
https://kamon.io/docs/latest/core/metrics/